modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -61,6 +61,7 @@ class AverageStatReport {
         this.count = 0;
         this.sum = 0;
         this.samples = [];
+        this.latestAvg = undefined;
```

Comment:
It's also used at line 254 in order to calculate the average RTT difference between the P2P connection and the JVB RTT (end-to-end).